specs/signing-and-verification-workflow.md

```diff
@@ -1,6 +1,6 @@
 # Signing and Verification Workflow
 
-This document describes how Notary v2 signs and verifies OCI artifacts.
+This document describes the workflow of signing and verifying OCI artifacts.
```

Comment:
images or artifacts? OCI artifact was changed in requirements/requirements.md to OCI image.

Reply:
In hti
notation can sign anything that's in a registry, which includes "images" as a specific type of artifact, and artifacts as a generic type (images, helm, opa, sboms, vex, scan reports, pulumi, ...)

Reply:
Thanks @TheFoxAtWork and @SteveLasker. The OCI image in the requirements/requirements.md means the OCI Image specification (there is a link associated with the text `OCI Image` and pointing to https://github.com/opencontainers/image-spec). The removed `OCI artifact` referred to specification https://github.com/opencontainers/artifacts, which was archived by OCI community recently. Both OCI Image and OCI artifact means the specifications in this context and defines how OCI artifacts are stored in the OCI compliant registries. 

To avoid confusion, I will update the sentence in requirements/requirements.md based on Steve's https://github.com/notaryproject/notaryproject/pull/262#discussion_r1272826213